contracts/v2/PolygonRollupManager.sol

```diff
@@ -347,15 +342,15 @@ contract PolygonRollupManager is
         IVerifierRollup verifier,
         uint64 forkID,
         VerifierType rollupVerifierType,
-        bytes32 genesis,
```

Comment:
This is ok to benamed genesis, maybe you wanted to update the addExistingRollup?